prestapyt/prestapyt.py

```diff
@@ -42,9 +45,10 @@
 try:  # for Python 3
     from http.client import HTTPConnection
 except ImportError:
-    from httplib import HTTPConnection
+    from http.client import HTTPConnection
```

Comment:
Nope, look at the content of the try except, this is specifically the python 2 import.
Note that normally `from http.client import HTTPConnection` should work even with py2 if python-futures is installed.